app/controllers/building_map_controller.rb

```diff
@@ -1,7 +1,7 @@
 class BuildingMapController < ApplicationController
   def index
-    @start = params[:start] if RoutingHelper.valid_coordinates?(params[:start])
-    @destination = params[:dest] if RoutingHelper.valid_coordinates?(params[:dest])
+    @start = RoutingHelper.coordinates(params[:start])
```

Comment:
This doesn't really tell me what this is doing, maybe something like `resolve_coordinates` or so would be clearer?